kong-0.5.0-1.rockspec

```diff
@@ -29,7 +29,8 @@ dependencies = {
 
   "luasocket ~> 2.0.2-5",
   "lrexlib-pcre ~> 2.7.2-1",
-  "lua-llthreads2 ~> 0.1.3-1"
+  "lua-llthreads2 ~> 0.1.3-1",
+  "sha1 ~> 0.5-1"
```

Comment:
Even i am using sha1 for test only otherwise i use ngx hmac-sha1function.
Question  is why to add library which is already available on luarocks?
On Sep 18, 2015 7:06 AM, "Thibault Charbonnier" <notifications@github.com>
wrote:

> In kong-0.5.0-1.rockspec
> <https://github.com/Mashape/kong/pull/548#discussion_r39857783>:
>
> > @@ -29,7 +29,8 @@ dependencies = {
> >
> >    "luasocket ~> 2.0.2-5",
> >    "lrexlib-pcre ~> 2.7.2-1",
> > -  "lua-llthreads2 ~> 0.1.3-1"
> > +  "lua-llthreads2 ~> 0.1.3-1",
> > +  "sha1 ~> 0.5-1"
>
> No, sha1 is a fallback for unit tests. Otherwise we use resty.string
> because it is included in OpenResty. However am considering luacrypto for
> the JWT plugin, would that be of any use for the HMAC? I suppose so.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/Mashape/kong/pull/548/files#r39857783>.
>